draft-pauly-add-resolver-discovery.md

```diff
@@ -200,6 +201,24 @@ Note that the domains listed in "trustedNames" may be broader than the zones tha
    }
 ~~~
 
+# Confirmation of Domain Name Ownership with TLS Certificates {#cert-name-check}
+
+A DoH server designation without DNSSEC or a PvD for mutual confirmation can be confirmed by the 
+SubjectAlternativeName field in the TLS certificate. When a client wants to confirm the validity of the 
+designation in this situation, it checks the TLS certificate of the DoH server for the name of the domain 
+which triggered the original designation query.
+
+The following example shows an HTTPS variant of the SVCB record type for "foo.example.com". If this record was
+received without DNSSEC, the client can confirm its validity by establishing a connection to "doh.example.net" 
+and checking the TLS certificate for the "foo.example.com" name. If the queried domain is not present in the TLS 
```

Comment:
What sort of check is this? LPM or exact match?

Reply:
I assume this is an exact match, right?

Reply:
Yeah, I think it should be. We should probably clarify that.

Reply:
Good question, Chris, I hadn't considered the ambiguity. I intended exact, and there's now a commit to that effect.

Reply:
Thanks!